libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/MatrixClient.kt

```diff
@@ -55,9 +55,15 @@ interface MatrixClient : Closeable {
      * Will close the client and delete the cache data.
      */
     suspend fun clearCache()
-    suspend fun logout()
+
+    /**
+     * Logout the user.
+     * Returns an optional URL. When the URL is there, it should be presented to the user after logout for RP initiated logout on their account page.
```

Comment:
Oh I see, this is what the URL was used for. The `RP` here is the client? I'm not too familiar with OpenID technical details.